Task 1/script.js

```diff
@@ -0,0 +1,39 @@
+/* ------------------------------ TASK 1 ----------------------------
+Parašykite JS kodą, kuris leis vartotojui įvesti svorį kilogramais ir
+pamatyti jo pateikto svorio kovertavimą į:
+1. Svarus (lb) | Formulė: lb = kg * 2.2046
+2. Gramus (g) | Formulė: g = kg / 0.0010000
+3. Uncijos (oz) | Formul4: oz = kg * 35.274
+
+Pastaba: atvaizdavimas turi būti matomas pateikus formą ir pateikiamas
+<div id="output"></div> viduje, bei turi turėti bent minimalų stilių;
+------------------------------------------------------------------- */
+
+const output = document.getElementById("output");
```

Comment:
Šaunu, kad tvarkingai pasirenkat elementus, bet būtų gerai žemiau esančioj logikoj pirmiausiai patikrit ar tas elementas egzistuoja tam atvejui, jei html struktūra pasikeistų be jūsų žinios. Taip išvengtumėte kodo lūžimo.